packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -575,6 +575,66 @@ export class PipelineEditor extends React.Component<
     });
   }
 
+  async handleOpenPipeline(): Promise<void> {
+    this.widgetContext.ready.then(() => {
+      let pipelineJson: any = this.widgetContext.model.toJSON();
+      const pipelineVersion: number = +Utils.getPipelineVersion(pipelineJson);
+      if (pipelineVersion !== PIPELINE_CURRENT_VERSION) {
+        // pipeline version and current version are divergent
+        if (pipelineVersion > PIPELINE_CURRENT_VERSION) {
+          // in this case, pipeline was last edited in a "more recent release" and
+          // the user should update his version of Elyra to consume the pipeline
+          showDialog({
+            title: 'Load pipeline failed!',
+            body: (
+              <p>
+                This pipeline corresponds to a more recent version of Elyra{' '}
+                <br />
```

Comment:
any reason for the <br /> mid sentence? depending on window/dialog size the formatting is off:

![image](https://user-images.githubusercontent.com/13156555/85306052-f275b980-b47b-11ea-9db2-48e242de9d12.png)

(same for the **Migrate pipeline?** dialog below (line 612))

Reply:
I guess it helps with a better visualization with the regular window size, but not much if you try to resize or do anything with the dialog. I will remove them.